src/Microsoft.AspNet.Mvc.Core/Formatters/JsonContractResolver.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Reflection;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.AspNet.Mvc
+{
+    /// <summary>
+    /// The default <see cref="IContractResolver"/> for <see cref="JsonInputFormatter"/> and 
+    /// <see cref="JsonOutputFormatter"/>. 
```

Comment:
why is it  needed on a `JsonOutputFormatter` we are not validating on the way out?